src/FsFhtw/Program.fs

```diff
@@ -1,6 +1,6 @@
 ﻿[<EntryPoint>]
 let main argv =
-    printfn "Welcome to the FHTW Domain REPL!"
+    printfn "Welcome to Texas Fold Them!"
```

Comment:
guter Punkt ... eventuell wollen wir das ganze Ding noch auf Holdem umbenennen? Das mit dem Fhtw irgendwas is noch nicht so nice